src/Installers/Windows/Wix.targets

```diff
@@ -2,7 +2,7 @@
   <!-- Set versioning properties after Arcade SDK targets have been imported. -->
   <PropertyGroup>
     <!-- Used for generating stable upgrade codes for bundles -->
-    <Version>$(AspNetCoreMajorVersion).$(AspNetCoreMinorVersion).$(AspNetCorePatchVersion).0</Version>
+    <Version>$(AspNetCoreMajorVersion).$(AspNetCoreMinorVersion).$(AspNetCorePatchVersion)$(_BuildNumberLabels)</Version>
```

Comment:
@wtgodbe is BuildNumberLabels a 2 part version? If so, this will turn Version into a 5 part number. That's fine for generating the GUID, however, the version property will be validated when the MSI is compiled and WiX may reject it I think.

Reply:
Yes, it is. I could replace it with `FileRevisionVersion` which is just the `20263` part. In that case though it wouldn't match the PackageVersion, is that alright? PackageVersion already contains `BuildNumberLabels` in non-stable builds

Reply:
Another option might be to just leave the Version as is, but include $(_BuildNumberLabels) in the <GuidInputs> so we end up with

```
<GuidInputs>$(Version);$(Platform);$(VersionSuffix);$(_BuildNumberLabels)</GuidInputs>
```